app/uk/gov/hmrc/voapropertylinking/connectors/modernised/ExternalPropertyLinkApi.scala

```diff
@@ -56,6 +56,10 @@ class ExternalPropertyLinkApi @Inject()(
         ).flatten
     )
 
+  def getMyOrganisationsPropertyLinksCount()(
```

Comment:
I wouldnt add this method yet, as modernised have not built it yet. in the service I would call the already used method for get my property links.